sources/packages/forms/src/form-definitions/studentprofile.json

```diff
@@ -1,5 +1,5 @@
 {
-  "title": "Student Profile",
+  "title": "Student Profile (New for Basic BCeID)",
```

Comment:
Can we delete the other 2 student profiles from DEV and name this one properly?
![image](https://user-images.githubusercontent.com/61259237/195940833-a17dbbca-e751-485e-bdaf-c8c59e70564f.png)

Reply:
@abschwenker  @guru-aot @dheepak-aot @henryoforeh-aot @andrepestana-aot i am deleting the first 2 files

Reply:
Can we remove the `(New for Basic BCeID)` now?